test/test-dockertools.js

```diff
@@ -155,96 +156,131 @@ describe('cloud-enablement:dockertools', () => {
-				.inDir(path.join(__dirname, './tmp'))
-				.withOptions({bluemix: JSON.stringify(scaffolderSampleSpring), platforms: ['cli'], buildType: 'maven'})
-		});
-
-		it('creates all docker and cli files', () => {
-			assert.file(['Dockerfile', '.dockerignore', 'Dockerfile-tools', 'cli-config.yml']);
-		});
-		it('Dockerfile contains app.jar', () => {
-			assert.fileContent('Dockerfile', '/app.jar');
-		});
-		it('.dockerignore does not contain wlp', () => {
-			assert.noFileContent('.dockerignore', 'wlp');
-		});
-		it('Dockerfile-tools does not contain wlp', () => {
-			assert.noFileContent('Dockerfile-tools', 'wlp/bin');
-		});
-	});
-
-	/* Full Content test for Java (Liberty) project */
-	describe('cloud-enablement:dockertools with Java-liberty project with buildType maven with platforms including cli', () => {
-		beforeEach(() => {
-			return helpers.run(path.join(__dirname, '../generators/app'))
-				.inDir(path.join(__dirname, './tmp'))
-				.withOptions({bluemix: JSON.stringify(scaffolderSampleJava), platforms: ['cli'], buildType: 'maven'})
-		});
-
-		it('creates all docker and cli files', () => {
-			assert.file(['Dockerfile', '.dockerignore', 'Dockerfile-tools', 'cli-config.yml']);
-		});
-		it('.dockerignore ignores workarea and logs', () => {
-			assert.fileContent('.dockerignore', 'workarea');
-			assert.fileContent('.dockerignore', 'logs');
-		});
-		it('Dockerfile contains installUtility', () => {
-			assert.fileContent('Dockerfile', 'installUtility');
-		});
-		it('Dockerfile-tools contains wlp path', () => {
-			assert.fileContent('Dockerfile-tools', 'wlp/bin');
-		});
-	});
-
-	/* Verify CLI tag behavior */
-	describe('cloud-enablement:dockertools with Java-liberty project with buildType maven with NO platforms', () => {
-		beforeEach(() => {
-			return helpers.run(path.join(__dirname, '../generators/app'))
-				.inDir(path.join(__dirname, './tmp'))
-				.withOptions({bluemix: JSON.stringify(scaffolderSampleJava), platforms: [], buildType: 'maven'})
-		});
-
-		it('creates a Dockerfile and .dockerignore', () => {
-			assert.file(['Dockerfile', '.dockerignore']);
-		});
-
-		it('does not create a Dockerfile-tools or cli-config.yml', () => {
-			assert.noFile(['Dockerfile-tools', 'cli-config.yml']);
-		});
-	});
+			
+			/* Verify CLI platform included */
+			describe('cloud-enablement:dockertools for ['+language+'] project using [' + buildType + '] (cli included)', () => {
+				let bluemixJson = language === 'SPRING' ? scaffolderSampleSpring : scaffolderSampleJava;
+				let options = {bluemix: JSON.stringify(bluemixJson), buildType: buildType, platforms: ['cli']};
+				beforeEach(() => {
+					return helpers.run(path.join(__dirname, '../generators/app'))
+						.inDir(path.join(__dirname, './tmp'))
+						.withOptions(options)
+				});
 
+				it('creates a Dockerfile and .dockerignore', () => {
+					assert.file(['Dockerfile', '.dockerignore', 'Dockerfile-tools', 'cli-config.yml']);
```

Comment:
The text content for this 'it' should read 'creates a Dockerfile, .dockerignore, Dockerfile-tools and cli-config.yml